src/Service.Host/client/src/components/BomVerificationHistoryUtility/BomVerificationHistory.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import {
+    InputField,
+    itemSelectorHelpers,
+    Loading,
+    Page,
+    SnackbarMessage
+} from '@linn-it/linn-form-components-library';
+
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import history from '../../history';
+import bomVerificationHistoryActions from '../../actions/bomVerificationHistoryActions';
+
+function BomVerificationHistory() {
+    const reduxDispatch = useDispatch();
+    const { id } = useParams();
+
+    const item = useSelector(reduxState =>
+        itemSelectorHelpers.getItem(reduxState.bomVerificationHistory)
+    );
+    const loading = useSelector(reduxState =>
+        itemSelectorHelpers.getItemLoading(reduxState.bomVerificationHistory)
+    );
+
+    const snackbarVisible = useSelector(reduxState =>
+        itemSelectorHelpers.getSnackbarVisible(reduxState.bomVerificationHistory)
+    );
+
+    useEffect(() => {
+        if (id) {
+            reduxDispatch(bomVerificationHistoryActions.fetch(id));
+        }
+    }, [id, reduxDispatch]);
+
+    return (
+        <Page history={history}>
+            {loading ? (
+                <Loading />
+            ) : (
+                <Grid container>
+                    <SnackbarMessage
+                        visible={snackbarVisible}
+                        onClose={() =>
+                            reduxDispatch(bomVerificationHistoryActions.setSnackbarVisible(false))
+                        }
+                        message="Create Successful"
+                        timeOut={3000}
+                    />
+                    <Grid item xs={12}>
+                        <Typography variant="h3">BOM Verification Entry</Typography>
+                    </Grid>
+                    <Grid item xs={12}>
+                        <InputField
+                            fullWidth
+                            value={`${item?.partNumber}`}
```

Comment:
you don't need the [`  template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here (and same for most of the other 'value' props for fields on these forms)